ui/src/plugins/com.google.PerfettoMcp/styles.scss

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 2025 The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.pf-ai-chat-panel {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
```

Comment:
Since the page PR https://github.com/google/perfetto/pull/2457 you are now wrapped in a container, which will scroll for you if you have a basic page, but you have a more complex page here where you want to place the input field at the bottom of the page.

The best solution is to just add `height: 100%` here. That'll force your `.pf-ai-chat-panel` element to fill the page container so you can manage the placement of the input footer manually and handle things like scrolling the conversation manually.